pytorch_object_detection/detection_models/__init__.py

```diff
@@ -68,7 +92,7 @@ def draw_boxes(imgs: List[Tensor], boxes: List[list], labels: List[List[str]]) -
         """
         images_with_boxes = []
         for ind, img in enumerate(imgs):
-            predicted_boxes = torch.Tensor(boxes[ind])
+            predicted_boxes = torch.Tensor(boxes[ind]).to(self.device)
```

Comment:
Why did you send boxes to GPU before drawing?

Reply:
let it be